ssv/spectest/tests/runner/preconsensus/invalid_msg_sig.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/bloxapp/ssv-spec/types/testingutils"
 )
 
-// InvalidMessageSignature tests SignedPartialSignatureMessage signature invalid
+// InvalidMessageSignature tests SignedPartialSignatureMessage signature invalid. No error is generated since the SignedPartialSignatureMessage.Signature is no longer checked
```

Comment:
```suggestion
// InvalidMessageSignature tests SignedPartialSignatureMessage signature invalid. No error is generated since the SignedPartialSignatureMessage. Signature is no longer checked.
// We wait for quorum to verify sig.
```

Reply:
The "no space" is intended. I wanted it to be: "SignedPartialSignatureMessage. Signature"